Doc/library/dis.rst

```diff
@@ -1086,6 +1079,22 @@ iterations of the loop.
    Pushes ``co_consts[consti]`` onto the stack.
 
 
+.. opcode:: LOAD_INT (i)
+
+   Pushes the integer ``i`` onto the stack.
+   ``i`` must be in ``range(256)``
```

Comment:
Why the restriction on the range of `I`? Can't use `EXTENDED_ARG` with this bytecode?

Reply:
We want to rapidly return a pre-allocated `int`, which is currently -5 to 257 (inclusive), as that is much faster than calling `PyLong_FromLong` or other API function.

Rather than worry about 257, is is simpler to just limit it to unextended instructions.

Reply:
Can we call it `LOAD_SMALL_INT`?

Reply:
We can. I have.